proposals/variant_set_metadata/README.md

```diff
@@ -0,0 +1,246 @@
+
+        }
+        "Carton_Opened" {
+           float3[] extentsHint = [(-6.27056, -6.53532, 0), (6.14027, 6.10374, 29.8274)]
+
+        }
+        "Carton_Sealed" {
+           float3[] extentsHint = [(-6.27056, -6.44992, 0), (6.14027, 6.10374, 29.2792)]
+
+        }
+    }
+    variantSet "shadingComplexity" = {
+        reorder variants = ["none", "display", "modeling", "full"]
+        "display" {
+
+        }
+        "full" {
+
+        }
+        "modeling" {
+
+        }
+        "none" {
+
+        }
+    }
+
+    variantSet "shadingTest" (
+        """Added UPC# for the display names"""
+        displayName = "Shading Test Variations"
+        doc = "Shading variations that are currently being explored for product placement"
+        variantDisplayNames = { "milkBrandA" : "Milk Brand A (UPC#: 123-ABC)", "milkBrandB" : "Milk Brand B (UPC#: 456-DEF)" }
+        hidden = True
+    ) = {
+        "milkBrandA" {
+
+        }
+        "milkBrandB" {
+
+        }
+    }
+}
+```
+
+## Variant Display Names
+
+Adding display names to variants is another metadata field that this proposal would like to support.
+This field will need to be handled differently than other displayName metadata fields 
+found on types like `UsdPrim` and `UsdProperty`. Specifically, a mapping of variant name
+to display name will be required as display name can not be added to a variant.
+First, there is no `UsdVariant` type to add display name API to. Secondly, and most importantly,
+all fields within a variant are applied directly to a `UsdPrim` when selected.
+If a displayName metadata field was added to a variant, that field would be applied to the
+`UsdPrim`'s displayName and not the variant's. It's for this reason that a mapping of 
+variant name to display name will be added to a `UsdVariantSet`.
+As described in [Metadata Value Resolution](#metadata-value-resolution), 
+the mapping of variant name to display name will need to be correctly resolved and merged.
+
+## Variant Ordering
+
+One of the metadata fields presented in this proposal is the 
+addition of [variantOrder](#metadata-fields). The goal of this field is simple;
+to specify an explicit order for how an editor should display variants in a `UsdVariantSet`.
+When considering the implementation of this metadata field for the `SdfTextFileFormat`
+it seems consistent with other `SdfSpec`s to use a `reorder variants` statement.
+This can be seen in the [proposed example](example.usda) but worth 
+calling out specifically as it brings up two questions:
+
+1. Is a `reorder variants` statement the correct approach for implementing `variantOrder` 
+    metadata field?
+2. If so, should the addition of a new `reorder` statement in the `SdfTextFileFormat` 
+    require a version bump as it requires changes to the parser?
+
+## Crate File Format Support
+
+The general consensus is that the USD Crate File Format should "just work" when 
```

Comment:
Yep, Crate shouldn't require any changes. It would just require defining what of the fields the variant set metadata gets added to on the underlying prim spec.